stanford_feeds_helper.module

```diff
@@ -69,13 +69,23 @@ function stanford_feeds_helper_feeds_set_target($source, &$entity, $target, $val
   $value = is_array($value) ? $value : array($value);
   $info = field_info_field($target);
 
+  // Sometimes courses have waaaaaaaay too many sections and we might want to
+  // restrict how many are being set because it breaks everyting. This is how.
+  $limit_collections = variable_get("stanford_feeds_importer_limit_field_collections", -1);
```

Comment:
Hey, do you want to name this variable the same thing that you're naming it here? https://github.com/SU-SWS/stanford_courses/pull/64/files#diff-4fe3d40ff019bf6d0ed90a8c3c2af745R481

`stanford_feeds_helper_limit_field_collections` vs. `stanford_feeds_importer_limit_field_collections`?